### DISCUSSION TITLE
Crash when open char bp from AAlsCharacter::PostInitializeComponents()

### DISCUSSION THREAD

**Discussion by Pooper333** [Sixze/ALS-Refactored#486] [Q&A]
I am using als v4 refactored and I get a crash everytime I open my character bp. I dont really mess with the c++ code at all, however, it crashed after I messed with the anim bp a little.  the error is on line 129 (GetMesh()->AddTickPrerequisiteActor(this);) 
here is the section of code:
```c++
void AAlsCharacter::PostInitializeComponents()
{
	// Make sure the mesh and animation blueprint are ticking after the character so they can access the most up-to-date character state.

	GetMesh()->AddTickPrerequisiteActor(this);

	AlsCharacterMovement->OnPhysicsRotation.AddUObject(this, &ThisClass::CharacterMovement_OnPhysicsRotation);

	// Pass current movement settings to the movement component.

	AlsCharacterMovement->SetMovementSettings(MovementSettings);

	AnimationInstance = Cast<UAlsAnimationInstance>(GetMesh()->GetAnimInstance());

	Super::PostInitializeComponents();
}
``` 

here is my crash log, any help is appreciated! 

1
Unhandled Exception: EXCEPTION_ACCESS_VIOLATION reading address 0x0000000000000000

UnrealEditor_ALS!AAlsCharacter::PostInitializeComponents() [C:\Users\G\Documents\Unreal Projects\ALSbattle\Plugins\ALS-Refactored-main\Source\ALS\Private\AlsCharacter.cpp:129]
UnrealEditor_Engine
UnrealEditor_Engine
UnrealEditor_Engine
UnrealEditor_Engine
UnrealEditor_Engine
UnrealEditor_UnrealEd
UnrealEditor_UnrealEd
UnrealEditor_UnrealEd
UnrealEditor_UnrealEd
UnrealEditor_UnrealEd
UnrealEditor_UnrealEd
UnrealEditor_UnrealEd
UnrealEditor_UnrealEd
UnrealEditor_UnrealEd
UnrealEditor
UnrealEditor
UnrealEditor
UnrealEditor
UnrealEditor
UnrealEditor
kernel32
ntdll

**Comment by ameaninglessname** [comment]
Which commit are you using?
It's feeling like a blueprint error, if so, recreating it might help.

**Reply by Pooper333** [reply]
I just used a backup of my character, so I guess my issue is fixed, however I think it was when I accidently changed something in the animbp that started the crashing, thanks for the help anyway!

**Reply by ameaninglessname** [reply]
Glad to hear that, you could mark this as resolved.
